_sass/screen.scss

```diff
@@ -5078,5 +5089,5 @@ html[data-theme="dark"] .dark-theme-toggle::before {
 //     @import 'dark-mode';
 // }
 // html[data-theme="dark"] {
-//     @import 'dark-mode'; 
```

Comment:
note this deletion...

Reply:
Thank you for your review.
I think my editor automatically deleted unneeded empty spaces and rows. It will make no effect functionally. Sorry to confuse you.